components/autocomplete/index.js

```diff
@@ -231,9 +232,15 @@ export class Autocomplete extends Component {
 		}
 	}
 
-	loadOptions( index ) {
-		this.props.completers[ index ].getOptions().then( ( options ) => {
-			const keyedOptions = map( options, ( option, i ) => ( { ...option, key: index + '-' + i } ) );
+	/**
+	 * Load options for an autocompleter.
+	 *
+	 * @param  {number} index    The autocompleter index.
```

Comment:
Nit: Spacing is misaligned with following `@param`, unnecessary second space after each `@param`

Reply:
fixed